lib/aurora_bootstrapper/utility.rb

```diff
@@ -0,0 +1,48 @@
+module AuroraBootstrapper
+  class Utility
+    def self.get_db_pass
+      db_pass = nil
+
+      if ENV.key?('DB_PASS')
+        # read from ENV directly
+        db_pass = ENV.fetch( 'DB_PASS')
+      else
+        file_object = File.open(ENV.fetch( 'DB_CRED_FILE' ))
+        # read from file stored in e.g. Vault etc
+        db_pass = file_object.read
+      end
```

Comment:
because ruby has implicit returns, you don't have to have `db_pass` at the end. `fetch` also takes an [optional block](https://apidock.com/ruby/Hash/fetch) which will fire if the key isn't present

```ruby
class Utils
  class << self
    def db_pass
      ENV.fetch( 'DB_PASS' ) do
        File.open( ENV.fetch( 'DB_CRED_FILE' ) ).read
      end
    end
  end
end
```

Reply:
oh shoot! missed this one

Reply:
ruby has lots of neat little tricks like that. it's hard to keep track of them all